app/views/layouts/_sidebar.html.slim

```diff
@@ -1,3 +1,4 @@
-div.sidebar.sidebar-left.sidebar-animate#site-navigation-sidebar class=sidebar_classes
-  ul.nav.navbar-stacked
-    = yield
+div class=sidebar_classes
+  nav.navbar-default.sidebar role='navigation'
+    div.collapse.navbar-collapse#site-navigation-sidebar
```

Comment:
This partial is not involved with `sidebar_items`. The current method for multiple sidebars is for the layout to handle it itself.

Only the `sidebar` helper method use this, which for now, is called at most once. Should I add that into the comments or?